sdk/program/src/pubkey.rs

```diff
@@ -992,4 +992,17 @@ mod tests {
         );
         assert!(pubkey_from_seed_by_marker(&PDA_MARKER[1..]).is_ok());
     }
+
+    /// 1111111QLbz7JHiBTspS962RLKV8GndWFwiEaqKM (before)
+    /// 1111111QLbz7JHiBTspS962RLKV8GndWFwiEaqKMxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx (this pr)
```

Comment:
comments don't really make sense post-merge. this kind of commentary belongs in pr description